core/chains/evm/txmgr/confirmer_test.go

```diff
@@ -1399,7 +1399,8 @@ func TestEthConfirmer_FindTxsRequiringRebroadcast(t *testing.T) {
 	etx1 := cltest.MustInsertUnconfirmedEthTxWithBroadcastLegacyAttempt(t, txStore, nonce, fromAddress)
 	nonce++
 	attempt1_1 := etx1.TxAttempts[0]
-	dbAttempt := txmgr.DbEthTxAttemptFromEthTxAttempt(&attempt1_1)
+	var dbAttempt txmgr.DbEthTxAttempt
```

Comment:
what's going on with all these? i can't figure out what the linter was upset about

Reply:
The funcs were used from for loops. Switching them to methods instead of passing pointers fixed some cases and was just more readable IMHO. I think they would have been methods originally, if it was valid to declare method on an alias type. I just reversed them to have the other concrete type as the receiver.